ramses_inspector.py

```diff
@@ -22,28 +23,30 @@ def __init__(self,
                  scene: ExportableScene):
 
         self.scene = scene
-        self._window = None
+        self.viewer_process = None
 
-    def load_viewer(self):
+    def load_viewer(self, platform, block: bool = False):
```

Comment:
tests are failing now - should add new platform argument there too

Reply:
Fixed the tests that were broken by this PR (and only them) in  436d33c